docs/source/tertiary/whole-genome-regression.rst

```diff
@@ -76,21 +75,23 @@ Example
 
     import pandas as pd
 
-    label_df = pd.read_csv(continuous_phenotypes_csv, index_col='sample_id') \
-        .apply(lambda x: x-x.mean())[['Continuous_Trait_1', 'Continuous_Trait_2']]
+    label_df = pd.read_csv(continuous_phenotypes_csv, index_col='sample_id')
+    label_df = ((label_df - label_df.mean())/label_df.std(ddof=0))[['Continuous_Trait_1', 'Continuous_Trait_2']]
```

Comment:
This is fine for now, but we might want to recommend that people just use [sciki-learn's standard scaler](https://scikit-learn.org/stable/modules/generated/sklearn.preprocessing.StandardScaler.html).

Reply:
Sure - I just didn't want to add an unnecessary dependency just for documentation purposes.